client/x/evmstaking/keeper/redelegation.go

```diff
@@ -15,6 +15,16 @@ import (
 )
 
 func (k Keeper) ProcessRedelegate(ctx context.Context, ev *bindings.IPTokenStakingRedelegate) error {
+	isInSingularity, err := k.IsSingularity(ctx)
+	if err != nil {
+		return errors.Wrap(err, "check if it is singularity")
+	}
+
+	if isInSingularity {
+		log.Info(ctx, "Relegation event detected, but it is not processed since current block is singularity")
```

Comment:
Should we change it to debug?

Reply:
Yes, changed in [2150314](https://github.com/piplabs/story/pull/202/commits/2150314811ef720fd1cf5ed4bb9e5ce7e5614da3)